modules/odc-using-web-terminal.adoc

```diff
@@ -18,4 +18,17 @@ After the Web Terminal Operator is installed, you can use the web terminal as fo
 +
 . Click *Start* to initialize the web terminal using the selected project.
 
++
```

Comment:
If you want the following comment aligned with the step on line 19, delete the blank line 20.